src/basic/linux/if.h

```diff
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0+ WITH Linux-syscall-note */
+/* SPDX-License-Identifier: GPL-2.0-or-later WITH Linux-syscall-note */
```

Comment:
these files are import from linux upstream, the spdx marker will be reset again when they are updated from upstream again

Reply:
Yeah, I was thinking that we might want to just ignore that part of the diff when applying. I wanted to have consistent spdx markers to make it easier to evaluate the whole tree using a script. But on second thought, updates would indeed be awkward. I'll drop that commit.